server/src/test/java/io/crate/planner/operators/PushDownTest.java

```diff
@@ -107,16 +107,15 @@ public void testOrderByOnJoinWithMultipleRelationsPushedDown() {
         LogicalPlan plan = plan("select t1.a, t2.b, t3.a from t1 " +
                                 "inner join t2 on t1.a = t2.b " +
                                 "inner join t1 as t3 on t3.a = t2.b " +
-                                "order by t2.b");
+                                "order by t1.a");
```

Comment:
ok, this is because the swap is gone and now the most left table is t1. This makes sense.

Reply:
The only thing I wonder is,why the test was with the table swapped there in the first place.

Reply:
@matriv git says you wrote the test in the first place. Any opinions here?  For me this change makes sense.

Reply:
@matriv thanks for the feedback.